list-credential.groovy

```diff
@@ -1,24 +1,31 @@
 /**
-Author: kuisathaverat
+Author: kuisathaverat, carlosrodlop
 Description: List ID and Description of all credentials on a Jenkins Instance.
+It includes the fingerprints under the following considerations: 
+   1) Job needs to have enable the Fingerprints. See https://www.jenkins.io/doc/book/using/fingerprints/ 
+   2) Usage tracking requires the cooperation of plugins and consequently may not track every use.
+Test on CloudBees CI Cloud Operations Center 2.277.1.2
 **/
 import com.cloudbees.plugins.credentials.Credentials
+import com.cloudbees.plugins.credentials.CredentialsProvider
 
-Set<Credentials> allCredentials = new HashSet<Credentials>();
-
-def creds = com.cloudbees.plugins.credentials.CredentialsProvider.lookupCredentials(
-      com.cloudbees.plugins.credentials.Credentials.class
-);
-
-allCredentials.addAll(creds)
+Set<Credentials> allCredentials = new HashSet<Credentials>()
```

Comment:
```suggestion
Set<Credentials> allCredentials = new HashSet<Credentials>();
```
so don't have any change here

Reply:
@fbelzunc, From my understanding when it compiles without semicolons that means that they are not needed https://dzone.com/articles/groovy-sometimes-you-still WDTY?